Tests/FluentAssertions.Equivalency.Specs/CollectionSpecs.cs

```diff
@@ -558,6 +558,425 @@ public void When_a_deeply_nested_property_of_a_collection_with_an_invalid_value_
             act.Should().NotThrow();
         }
 
+        public class For
+        {
+            [Fact]
+            public void For_Does_Not_Exclude()
```

Comment:
🔧 This does not follow our naming convention.

Reply:
Removed test since it does not compile anymore